### PR TITLE
OCLOMRS-264: Refactor the navbar items bold styling to avoid repetition of strong tags

### PR DESCRIPTION
**Pull request by Duncanian**
# JIRA TICKET NAME:
[Refactor the navbar items bold styling to avoid repetition of strong tags](https://issues.openmrs.org/browse/OCLOMRS-264)

# Summary:
IInitially, strong tags were used to make the navbar items to become bold. This led to the repetition of the code hence the styling will now be in sass thus increasing reusability and decreasing repetition of code.

### CODE REVIEW

File: src/styles/app.scss [hunk 1]
```diff
@@ -19,6 +19,7 @@
   padding: 0rem 0rem;
   padding-right: .5rem !important;
   padding-left: .5rem !important;
+  font-weight: 700;
```
Comment (dondrzzy) [inline src/styles/app.scss]: @Duncanian, I hope you realized that the logo or anchor tag with a class of `navbar-brand` had strong tags but has no class of `nav-link`.

**Comment by Michyjones** [comment]
@Duncanian fix this conflict

File: src/styles/app.scss (hunk 1 above)
Comment (Duncanian) [reply src/styles/app.scss]: Let me add it ASAP

Comment (Duncanian) [reply src/styles/app.scss]: Done.

**Comment by Duncanian** [comment]
@Michyjones , I have fixed it.

**Pull request merged**